docs/deploy.md

```diff
@@ -38,12 +38,14 @@ Make sure that your Kubernetes config (e.g, `~/.kube/config`) is pointing to the
    of the Istio Ingress Gateway service. You can retrieve the external IP of this service by running
    `kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[*].ip}'`
    1. If you used the `./hack/generate-values.sh` script then you should only
-      configure a single DNS record for the domain you passed as input to the
-      script and have it resolve to the Ingress Gateway's external IP
+      configure a single DNS record for the wildcard subdomain of the domain
+      you passed as input to the script and have it resolve to the Ingress
+      Gateway's external IP
 
 ## Validate the deployment
 
-1. Set up cf cli to point to CF:
+1. Get the `<cf_admin_password>` with the `bosh interpolate /tmp/cf-values.yml --path /cf_admin_password`
+command and set up cf CLI to point to CF:
    ```bash
    $ cf api --skip-ssl-validation https://api.<system_domain>
    $ cf auth admin <cf_admin_password>
```

Comment:
could you set this line to be `cf auth admin $(bosh interpolate /tmp/cf-values.yml --path /cf_admin_password)` and reduce the need to copy/paste or type ?

Reply:
I agree @paulczar, that's what I had in my original PR and was asked to switch it to this! Happy to PR whatever the maintainers collectively agree upon, my vote is for the same you're suggesting. Please let me know what y'all agree to.